schemas/FollowUp.json

```diff
@@ -30,7 +30,8 @@
     }},
     "deviceId": { "type": "string" },
     "dateOfVisit": { "type": "string", "format": "date-time" },
-    "personId": { "type": "string" }
+    "personId": { "type": "string" },
+    "dispatcherNotification": { "type": "string", "pattern": "^started$" }
```

Comment:
I wanted to keep it minimal, as for now i am going to store just a "started" value. following your suggestion i made it an object in order to easier allow possible extensions